json-formatter/javascript/features/step_definitions/steps.ts

```diff
@@ -8,6 +8,10 @@ Given('a passed {word} with text attachment {string}', function(word: string, at
   this.attach(attachmentText, 'text/plain')
 })
 
+Given('a step outputs:', function(txt: string) {
+  this.attach(txt, 'text/output')
```

Comment:
`text/output` is not a valid media type. It should be `text/plain`. Alternatively, it should be an [extension](https://en.m.wikipedia.org/wiki/Media_type) - I suggest `text/x.cucumber.output+plain`.

Reply:
sounds fine, I was definitely not convinced by `text/output` anyway, sounded too hacky :D